adr/0009-frontend-directory-structure-and-architecture.md

```diff
@@ -0,0 +1,64 @@
+# 9. Frontend directory structure and architecture
+
+Date: 2023-05-16
+
+## Status
+
+Accepted
+
+## Context
+
+Our software project has a frontend that needs to be organized to facilitate maintenance, scalability, modularity, and ease of navigation. The current directory structure is outdated and hard to manage. As software engineers, we propose a new directory structure based on some criteria.
+
+## Decision
+
+We propose the following new directory structure for the frontend of our software project:
+
+```
+assets/js
+├── app.js
+├── common
```

Comment:
Maybe it would be a good idea to define what this `common` holds.
According redux docs:
```
/common contains truly generic and reusable utilities and components.
```
Maybe we can put this sentence as paragraph below, so we have a written and aknowledged agreement hehe

Reply:
Yes you're totally right :+1: